cognite/neat/_rules/models/_base_rules.py

```diff
@@ -326,19 +328,20 @@ def dump(
             exclude_unset: Whether to exclude fields that have not been explicitly set.
             exclude_defaults: Whether to exclude fields that are set to their default value.
         """
-        for field_name in self.model_fields.keys():
-            value = getattr(self, field_name)
-            # Ensure deterministic order of properties, classes, views, and so on
-            if isinstance(value, SheetList):
-                value.sort(key=lambda x: x._identifier())
+        if sort:
+            for field_name in self.model_fields.keys():
+                value = getattr(self, field_name)
+                # Ensure deterministic order of properties, classes, views, and so on
+                if isinstance(value, SheetList):
+                    value.sort(key=lambda x: x._identifier())
```

Comment:
Introduced this as a parameter and default to not sorting. The motivation is that when dumping a solution model is that you want to have a special order of properties such that the 'Enterprise' view properties is next to the one that points to it.